add-item.html

```diff
@@ -2,16 +2,234 @@
+                                    <label for="radio__button5">Latin American</label>
+                                </div>
+
+                                <div class="radio__button">
+                                    <input type="radio" name="race" id="radio__button6" value="other" required />
+                                    <label for="radio__button6">Other</label>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="checkbox">
+                            <h3>Choose everything that applies.</h3>
+
+                            <div class="checkbox__options">
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option1" value="student" />
+                                    <label for="checkbox__option1">Student</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option2" value="teacher" />
+                                    <label for="checkbox__option2">Teacher</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option3" value="businessman" />
+                                    <label for="checkbox__option3">Business owner</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option4" value="freelancer" />
+                                    <label for="checkbox__option4">Freelancer</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option5" value="traveller" />
+                                    <label for="checkbox__option5">Traveller</label>
+                                </div>
+
+                                <div class="checkbox__option">
+                                    <input type="checkbox" name="way" id="checkbox__option6" value="tourist" />
+                                    <label for="checkbox__option6">Tourist</label>
+                                </div>
+
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="select__date">
+
+                        <h3>When do you plan to travel?</h3>
+                        <div class="date">
+                            <input type="date" name="startDate" id="travel_date" required />
+                            <input type="date" name="endDate" id="travel_date" required />
```

Comment:
id should always be unique, please look into the link below:
https://www.w3schools.com/html/html_id.asp

Reply:
solved.
``` 
<input type="date" name="startDate" id="travel_date1" required />
                            <input type="date" name="endDate" id="travel_date2" required />
```